CabalHelper/Wrapper.hs

```diff
@@ -59,7 +59,7 @@ usage = do
 \  [--with-cabal=CABAL_PATH]\n\
 \  [--with-cabal-version=VERSION]\n\
 \  [--with-cabal-pkg-db=PKG_DB]\n\
-\  PROJ_DIR DIST_DIR ( print-exe | package-id | [CABAL_HELPER_ARGS...] ) )\n"
+\  PROJ_DIR DIST_DIR ( print-exe | print-exe-prepared | package-id | [CABAL_HELPER_ARGS...] ) )\n"
```

Comment:
I think this goes over 80 cols please split the line and align the pipes to the opening paren.